app/views/check_your_answers/show.html.erb

```diff
@@ -3,24 +3,15 @@
 
     <h1 class="govuk-heading-xl"><%= t('application_form.check_your_answers') %></h1>
 
-    <h2 class="govuk-heading-m"><%= t('application_form.personal_details_section.heading') %></h2>
+    <%= render 'shared/section_summary',
+      fields: [:first_name, :last_name, :preferred_name, :date_of_birth, :nationality],
+      section_name: :personal_details,
+      model: @personal_details %>
```

Comment:
I used the same paras `model` here, but I realized that `:model` should refer to `PersonalDetail` and not to the instance `@personal_details`, that is the instance. 
It might be worth to rename it to `:entity`

Reply:
:+1: We're going to rename it to `record`

Reply:
It has been renamed to `record`